toolkit/tools/pkg/imagecustomizerlib/imagecustomizer.go

```diff
@@ -301,15 +301,15 @@ func customizeVerityImageHelper(buildDir string, baseConfigPath string, config *
 	saltMatches := saltRegex.FindStringSubmatch(verityOutput)
 	if len(saltMatches) > 1 {
```

Comment:
Write it like this:

```golang
if len(saltMatches) <= 1 {
    return fmt.Errorf("failed to parse salt from veritysetup output")
}

salt = saltMatches[1]
```

See, https://llvm.org/docs/CodingStandards.html#use-early-exits-and-continue-to-simplify-code